app/scripts/metamask-controller.js

```diff
@@ -761,8 +761,6 @@ export default class MetamaskController extends EventEmitter {
         }),
     });
 
-    this.nftController.setApiKey(process.env.OPENSEA_KEY);
```

Comment:
This was still being set, but it did nothing: https://github.com/MetaMask/core/blob/f8e5f2e3cadecc44c1c5b40b3ed3c36d154346cb/packages/assets-controllers/src/NftController.ts#L1382

(see that the key isn't used anywhere).